app/controllers/v0/backend_statuses_controller.rb

```diff
@@ -2,8 +2,10 @@
 
 module V0
   class BackendStatusesController < ApplicationController
-    skip_before_action :authenticate, only: [:show]
+    skip_before_action :authenticate, only: %i[index show]
 
+    # Note: this endpoint is somewhat misleading.  Index gets data from PagerDuty and
+    # show only looks at GI bill scheduled downtime (and gets no data from PagerDuty)
```

Comment:
[Here's](https://github.com/department-of-veterans-affairs/vets-api/pull/2905#issuecomment-475626457) how we ended up in this place.  Agreed that `show` should be refactored in the future to exist more cohesively with `index` functionality.